mozjs/src/generate_wrappers.sh

```diff
@@ -27,12 +27,15 @@ grep_heur() {
   grep -v 'MutableHandleObjectVector' # GetDebuggeeGlobals has it
 }
 
-# clone file and reformat
-cp target/debug/build/mozjs_sys-*/out/build/jsapi.rs target/jsapi.rs
-rustfmt target/jsapi.rs --config max_width=1000
-cp target/debug/build/mozjs-*/out/gluebindings.rs target/glue.rs
-rustfmt target/glue.rs --config max_width=1000
+find_fmt_parse() {
```

Comment:
How about `find_latest_version_of_file_parse_and_format`?

Reply:
That is so loooooong. Maybe just `last_file_parse`?

Reply:
Bits are very cheap. It is better to be verbose and obvious, rather than short and obscure. Many people will read this code in the future and it took me a bit to understand it.

Reply:
used long name and added usage comment and some more comments inside function